crates/pessimistic-proof-core/src/multi_batch_header.rs

```diff
@@ -27,6 +27,11 @@ where
 {
     /// Network that emitted this [`MultiBatchHeader`].
     pub origin_network: NetworkId,
+    /// Current height of the L2 chain.
+    pub height: u64,
```

Comment:
It is not the L2 chain height (this can refer to the number of blocks) while here, it is the height of the network in regard of the agglayer based on how many certificates it receives

Reply:
yeah misleading indeed, what about this:

```diff
-    /// Current height of the L2 chain.
+    /// Current certificate height of the L2 chain.
```

also related thread:  https://github.com/agglayer/agglayer/pull/689#discussion_r1987575045

Reply:
This is fine solution for me